clients/stellar-relay-lib/src/config.rs

```diff
@@ -39,6 +39,10 @@ impl StellarOverlayConfig {
 		let cfg = &self.connection_info;
 		let secret_key = SecretKey::from_encoding(secret_key)?;
 
+		let public_key = secret_key.get_public().to_encoding();
+		let public_key = std::str::from_utf8(&public_key).unwrap();
+		log::info!("CONNECTED TO KEY: {public_key}");
```

Comment:
Let's give more context in this comment. Something like, 
"Connected to Stellar overlay network with public key:  {public_key}"